ui/src/app/tags/components/DefinitionField/DefinitionField.tsx

```diff
@@ -85,12 +86,14 @@ export const DefinitionField = ({ id }: Props): JSX.Element => {
 //node[@id="firmware"]/capabilities/capability/@id = "uefi"`}
         rows={3}
       />
-      {/* // TODO: Add the link to the docs:
-          // https://github.com/canonical-web-and-design/app-tribe/issues/748 */}
       <p className="p-form-help-text u-sv1">
         Add an XPath expression as a definition. MAAS will auto-assign this tag
         to all current and future machines that match this definition. More
-        about how to use <a href="#todo">XPath Expression</a>.
+        about how to use{" "}
+        <a href="https://maas.io/docs/how-to-work-with-tags#heading--xpath-expressions">
```

Comment:
Same here. Also, the link should be something like "Learn more about..." not just More about.